app/styles/components/_nav.scss

```diff
@@ -116,3 +118,9 @@ $nav-burger-height: 16px;
     display: none;
   }
 }
+
+@media (min-width: $nav-break-mobile+1) {
```

Comment:
I noticed that when the window width is equal to $nav-break-mobile the burger disappears. This way it behaves correctly.